service/service-samples/pom.xml

```diff
@@ -41,6 +41,7 @@
         <module>sample-service-rest-cassandra-book</module>
         <module>sample-service-rest-mysql-book</module>
         <module>sample-service-rest-oracle-book</module>
+<!--        <module>sample-service-rest-native-book</module>-->
```

Comment:
commented out?

Reply:
ignore samples. this module still exists, the pom needs to be corrected as native image is failing to build.